wordle-web/plugins/locally.js

```diff
@@ -0,0 +1,21 @@
+export default defineNuxtPlugin(() => {
```

Comment:
Is this another implementation of the nuxt-storage package shown in class?

Reply:
Yes. I think this is when we were adding in beforehand but don't know for sure. I can ask Alexa about it since she's the one that added that part in.